houses.ipynb

```diff
@@ -2025,6 +2063,1315 @@
     "with pd.option_context('display.max_rows', None, 'display.max_columns', None): \n",
     "    print(description)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Brakujące dane\n",
+    "\n",
+    "Widać, że w niektórych przypadku niektórych cech występuje brak danych. Usuwamy te cechy (np. rok wybudowania garażu.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 228,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data = train_data.dropna(axis=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Histogramy"
```

Comment:
Would it be possible to have boxplots next to histograms for every feature, in one line? It would be much more readable
If not, I think it should be consistent for plots before and after cleaning, I prefer the second version (histogram then boxplot, not all histograms then all boxplots)

Reply:
Sure, that would make sense :+1: